config/settings/test.yml

```diff
@@ -429,6 +429,7 @@ travel_pay:
     resource: 'resource_id'
     tenant_id: 'tenant_id'
     auth_url: 'https://auth.veis.gov'
+  mock: false
```

Comment:
Is there any way this can be `true` and still exhibit the behavior we want for the tests? I was thinking about it, and if we miss a `VCR.use_cassette`, this'll will let a real call to the API go through right?

Reply:
Initally thought that these had to be `false` to use VCR, but turns out that I can set this to `true` and still use the `VCR` cassettes.